### PR TITLE
chore: add meetup link icon in the footer

### LINKED ISSUE #1434: ✨ [Feature request]: add our meetup link icon in the footer
### Is your feature request related to a problem? Please describe.

add the meetup link icon & link  to the footer of the reactplay website 

meetup link: https://www.meetup.com/reactplay-bengaluru/

### Describe the solution you'd like.

.

### Describe alternatives you've considered.

.

### Screenshot / Screenshare

image

### Additional context

_No response_

### Record

- [X] I have checked the existing [issues](https://github.com/reactplay/react-play/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/reactplay/react-play/blob/main/CONTRIBUTING.md)
- [ ] I want to work on this issue

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Meetverma**
# Description

Added a meetup link button in the footer

Fixes #1434

## Type of change
- [Y] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested locally and Works Fine!

# Checklist:

- [Y] I have performed a self-review of my own code
- [Y] My changes generate no new warnings
- [Y] New and existing unit tests pass locally with my changes

# Screenshots or example output
Screenshot 2024-02-07 at 7 12 35 PM

### CODE REVIEW

**Comment by priyankarpal** [comment]
Hello @Meetverma, please refrain from making these types of commits in the future

header

File: src/plays/quiz-app/QuizApp.tsx
Comment (priyankarpal) [inline src/plays/quiz-app/QuizApp.tsx:1]: please fix the formatting issue that's why the building is failing

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by atapas — APPROVED** [review]

File: src/plays/quiz-app/QuizApp.tsx
Comment (priyankarpal) [reply src/plays/quiz-app/QuizApp.tsx:1]: @Meetverma i will suggest to create new branch for PRs avoid to push from your main branch

Comment (priyankarpal) [reply src/plays/quiz-app/QuizApp.tsx:1]: for now it's ok but in future create a different branch & the name should be like this `fix-issue-issue-number`

**Review by priyankarpal — APPROVED** [review]

**Comment by Meetverma** [comment]
Thanks @priyankarpal 
That Was my first commit to react-play...
Looking forward for more contributions :)

**Pull request merged**